core-bundle/src/Entity/RememberMe.php

```diff
@@ -43,14 +43,14 @@ class RememberMe
     /**
      * @var string
      *
-     * @ORM\Column(type="string", length=64, nullable=false, options={"fixed"=true})
+     * @ORM\Column(type="binary_string", length=32, options={"fixed"=true})
```

Comment:
The value cannot be `null`, why did you remove `nullable=false`?

Reply:
Re-added in fc54f7bcb462656f1042b2ad01fc1d920cd60208.